guilded/http.py

```diff
@@ -141,9 +144,6 @@ async def login(self, email, password):
         data = await self.request(Route('GET', '/me'))
         return data
 
-    async def logout(self):
-        return await self.request(Route('POST', '/logout'))
-
```

Comment:
Breaks `Client.close`. Why would you remove logout anyway?

Reply:
It was declared twice, i removed duplicate